Steer/DigitizerWorkflow/src/SimpleDigitizerWorkflow.cxx

```diff
@@ -739,7 +739,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   }
   // GRP updater: must come after all detectors since requires their list
```

Comment:
@sawenzel fixing the conflict after rebasing your PR I see that the CTP specs do not follow the same    digitizerSpecs/ writerSpecs pattern, was it intended? https://github.com/AliceO2Group/AliceO2/pull/8718/commits/2175faeaa7968107b31aa1f6eaa713977993337c#diff-b1fedbaacd6c752678b71e5f22d308e35c2c3c8d4b9841013d900ba4faffdc7dR736-R739